.github/workflows/cicd_pipeline.yml

```diff
@@ -46,6 +46,15 @@ jobs:
     with:
       sha: ${{ github.event.pull_request.head.sha || github.event.after }}
 
+  run_functional:
+    name: "Tests"
+    if: github.event_name == 'push' || github.event.pull_request.draft == false
+    uses: heartexlabs/label-studio-frontend/.github/workflows/fun_tests.yml@fb-lsdv-4734/cypress-init
```

Comment:
Don't forget to change branch to `master` before merge

Reply:
sure!